src/js/rx/containers/Active.jsx

```diff
@@ -22,6 +24,15 @@ class Active extends React.Component {
   constructor(props) {
     super(props);
     this.handleSort = this.handleSort.bind(this);
+
+    this.checkWindowSize = _.debounce(() => {
+      if (document.documentElement.clientWidth < 600) {
```

Comment:
Should we put a comment here? It is 600 because the `$small` breakpoint from foundation is set at `40em`, and our base font-size is 15px